snippets/language-asciidoc.cson

```diff
@@ -250,7 +255,7 @@
     descriptionMoreURL: 'http://asciidoctor.org/docs/user-manual/#revision-number-date-and-remark'
     prefix: 'rev'
     body: '''
-      v1.0, ${3:revdate}: ${4:revremark}
+      v1.0, ${1:revdate}: ${2:revremark}
```

Comment:
I'd like to include the revision number in the placeholder steps. I'm not sure we should include revremark by default. This very rarely used in practice.

```
body: '''
v${1:1.0.0}, ${2:date}
$0
'''
```